README.md

```diff
@@ -31,7 +39,46 @@ env:
 ```
 Please refer to our [troubleshooting section](docs/DEVELOPER_GUIDE.md#troubleshooting) for recommendations and tips.
 
-### Use ###
+### KFServing in 5 Minutes
+Make sure you have [kind](https://github.com/kubernetes-sigs/kind#installation-and-usage),
```

Comment:
Why is this necessary?

Reply:
Can we at least make this optional (like a dropdown section) for those without a cluster.